ubuntu_image/builder.py

```diff
@@ -297,8 +302,10 @@ def prepare_filesystems(self):
             raise AssertionError('No room for root filesystem data')
         self.rootfs_size = avail_space
         self.root_img = os.path.join(self.images, 'root.img')
-        run('dd if=/dev/zero of={} count=0 bs={}M seek=1'.format(
-            self.root_img, avail_space))
+        # Create empty file with holes.
```

Comment:
Suggesting changes to my own writing here :) But maybe `Create sparse file` to use a more matching technical term?